refinery/tool_manager/models.py

```diff
@@ -11,27 +11,32 @@
 from django.http import JsonResponse
 
 import bioblend
-from bioblend.galaxy.dataset_collections import (CollectionDescription,
-                                                 CollectionElement,
-                                                 HistoryDatasetElement)
-from constants import UUID_RE
-from django_docker_engine.container_managers.docker_engine import \
+from bioblend.galaxy.dataset_collections import (
+    CollectionDescription, CollectionElement, HistoryDatasetElement
+)
+from constants import REFINERY_SOLR_DOC_LIMIT, UUID_RE
```

Comment:
import constants and move it into the "local" import section below since this is not a 3rd party module